eda-requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas
+matplotlib
+numpy
+seaborn
+cv2
+sklearn
+rasterio
```

Comment:
Pin the versions? Maybe we can move the installation process to the notebook itself? We don't want to end up with 20x reuirements.txt :)